jenkins/scripts/integration_delete.sh

```diff
@@ -15,10 +15,18 @@ CI_DIR="$(dirname "$(readlink -f "${0}")")"
 # shellcheck disable=SC1090
 source "${CI_DIR}/utils.sh"
 
+VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
+TEST_EXECUTER_VOLUME_NAME="${TEST_EXECUTER_VOLUME_NAME:-ci-test-volume-${VM_TIMELABEL}}"
```

Comment:
This is problematic, because the VM_TIMELABEL variable is actually not defined in the pipeline. so your name would mismatch. You should instead use the same approach as for the vm name (if not directly using the vm name instead of a volume name) : https://github.com/metal3-io/project-infra/blob/master/jenkins/jobs/integration_tests.pipeline#L49

Reply:
I will directly use the vm name instead of declaring and using another name for a volume